lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -660,3 +741,12 @@ def _try_get_node_ip_address(
         addr = addrs[0]["addr"]
         assert isinstance(addr, str)
         return addr
+
+    def _get_libvirt_conn_string(self) -> str:
+        hypervisor = "qemu"
+        host = self.qemu_platform_runbook.host
+
+        host_addr = host.address if host is not None else ""
+        transport = "+tcp" if host is not None else ""
```

Comment:
It looks like the `tcp` protocol is unencrypted. Is it possible to use `ssh` instead?

Reply:
I will investigate it. `tcp` seemed the simplest to use so I just went ahead with it to get things working quickly...

Reply:
Please at least try this, as we are already using `ssh` so it should just work. However, if there is unexpected complexity, then feel free to fix this in a subsequent task.

Reply:
I tried it out. It's not straightforward. There is no way to tell libvirt to use a specific ssh key. It picks up the config from ssh config file (e.g. `~/.ssh/config`). So we would need to modify that config programmatically and also clean it up later...

Does it really matter that the connection is unencrypted? Since this would only be used in a test environment...

Reply:
I think I found the docs on how to specify the SSH key: https://libvirt.org/uri.html#URI_remote. You can add a `?keyfile=` option to the end of the URL. Also, it probably make sense to use `+libssh2` instead of `+ssh` to improve performance.

It is generally a good idea to encrypt everything nowadays regardless of what it is being used for or where it is being run.

Reply:
Changed it to ssh. libssh didn't work. It said "libssh transport support was not enabled". Looks like it is a compile time flag that needs to be enabled while building libvirt.